### PR TITLE
Toggles facet button on mobile depending on page

### LINKED ISSUE #76: On document show page, facets disappear
Should we disable the slide out window on this page or figure out some way to render the facets

### PR DESCRIPTION
**Pull request by straleyb**
Fixes #76

### CODE REVIEW

File: app/assets/javascripts/slide_button_toggle.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
```
Comment (terrellt) [inline app/assets/javascripts/slide_button_toggle.js]: Shouldn't this toggle based on whether or not the facet ID exists?

Comment (straleyb) [inline app/assets/javascripts/slide_button_toggle.js]: So I changed it to check to see if 'oregondigital:' is in the path, which should mean that the id exists. I don't even know why I didn't check that the id existed in the first place. :+1:

**Comment by straleyb** [comment]
The Facet ID haha. I gotcha. :+1:

**Comment by terrellt** [comment]
Needs rebased again, and previous rebase left out the mock data file.

**Comment by straleyb** [comment]
@terrellt Ok this is finally rebased

**Pull request merged**